superdesk/vocabularies/vocabularies.py

```diff
@@ -236,10 +236,10 @@ def get_rightsinfo(self, item):
             return {}
 
     def get_extra_fields(self):
-        return list(self.get(req=None, lookup={'field_type': {'$exists': True}}))
+        return list(self.get(req=None, lookup={'field_type': {'$exists': True, '$ne': None}}))
```

Comment:
imo `$ne: None` should be enough, such field must exist

Reply:
Yes, true